ansible_collections/arista/avd/plugins/plugin_utils/cv_client/workflows/verify_devices_on_cv.py

```diff
@@ -22,11 +24,59 @@ async def verify_devices_on_cv(devices: list[CVDevice], workspace_id: str, skip_
     if not devices:
         return
 
+    # Finding entries with same serialNum or same mac address
```

Comment:
Move all this to a function. Maybe it can be shared with the check of the data from CV.